rollbar/contrib/django/middleware.py

```diff
@@ -276,6 +276,13 @@ def process_response(self, request, response):
     def process_exception(self, request, exc):
         if isinstance(exc, Http404) and _should_ignore_404(request.get_full_path()):
             return
+        if request.sensitive_post_parameters:
```

Comment:
Use if request.method == "POST" to see if request contains method POST: https://docs.djangoproject.com/en/4.0/ref/request-response/#django.http.HttpRequest.POST

Reply:
This is going to run per each request and wondering if you can use following instead of COPY()
mutable = request.POST._mutable
request.POST._mutable = true
Update the param
request.POST._mutable = mutable

Reply:
> Use if request.method == "POST" to see if request contains method POST: https://docs.djangoproject.com/en/4.0/ref/request-response/#django.http.HttpRequest.POST

I don't think you really need to do that, because I am checking if sensitive_post_parameters is not empty and then check if param is in request.POST. Not sure if extra if statement is needed here.

Reply:
> Instead of copy/paste the code, I would suggest to create a function to call.

yeah, I was thinking about it, I will do that

Reply:
@nahrinrollbar , your comments have been addressed. thx